recipes/wasabi/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "wasabi" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a2ba97e2d960a25cba876806501f2dc68f74470ccef324c0d7aaf82a25a7422 
+build:
+  noarch: python
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+    requires:
+        - pytest <4
+    imports:
+        - wasabi
+
+about:
+  home: http://github.com/ines/wasabi
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A lightweight console printing and formatting toolkit'
+
+  # The remaining entries in this section are optional, but recommended
+  dev_url: https://github.com/ines/wasabi
```

Maybe considering adding the `doc_url` for completeness?
```suggestion
  dev_url: https://github.com/ines/wasabi
  doc_url: https://github.com/ines/wasabi#-api
```